sources/packages/web/src/components/common/applicationTracker/InProgress.vue

```diff
@@ -52,33 +52,54 @@
     ></application-status-tracker-banner
   >
 
-  <application-status-tracker-banner
-    label="Waiting for additional information from a parent"
-    icon="fa:fas fa-clock"
-    icon-color="secondary"
-    v-if="applicationDetails?.parent1Info === SuccessWaitingStatus.Waiting"
-    ><template #content
-      >We are waiting for
-      <strong>supporting information from a parent.</strong> Please check your
-      email to confirm that you have received a message from us. This email will
-      include important details and links that your parent will need in order to
-      provide their information for your application.</template
-    ></application-status-tracker-banner
+  <template
+    v-for="(parent, index) in applicationDetails?.parentsInfo"
```

Comment:
I did not see a need to use an index when the object has a unique property. Can it be changed as below?
```vue
v-for="parent in applicationDetails?.parentsInfo" :key="parent.supportingUserId"
```